dronekit/__init__.py

```diff
@@ -1157,6 +1157,16 @@ def set_rc(chnum, v):
             set_rc(6, m.chan6_raw)
             set_rc(7, m.chan7_raw)
             set_rc(8, m.chan8_raw)
+            set_rc(9, m.chan9_raw)
```

Comment:
This could be done with a loop and `set_rc(i, m.getattr("chan%u_raw", i))` - but fine as-is.